ui/app/pages/swaps/build-quote/build-quote.js

```diff
@@ -119,9 +132,9 @@ export default function BuildQuote({
   const selectedToToken =
     tokensToSearch.find(({ address }) => address === toToken?.address) ||
     toToken;
-  const toTokenIsNotEth =
+  const toTokenIsNotDefault =
     selectedToToken?.address &&
```

Comment:
I don't think we need this check because `isSwapsDefaultTokenAddress` inherently would return `false`

Reply:
right, but line 137 is "**_!_**isSwapsDefaultTokenAddress(selectedToToken?.address, chainId)", so when `selectedToToken?.address` is a falsy value, `toTokenIsNotDefault` is true, _because of the `!`_